cmd/kubelet/app/options/options.go

```diff
@@ -114,6 +112,7 @@ func NewKubeletServer() *KubeletServer {
 			RktPath:                        "",
 			RktStage1Image:                 "",
 			RootDirectory:                  defaultRootDir,
+			RuntimeContainer:               "/docker-daemon",
```

Comment:
The issue right now is that there is no easy way for people to restrict the kubelet from not moving docker daemon into `/docker-daemon` cgroups. This PR fixes that in addition to fixing the metrics integration.
Magical names are much harder to manager. If we are going with default cgroup hierarchies, I'd prefer getting rid of all the other flags that let users specify cgroups and let kubelet decide defaults - be it systemd or non-systemd nodes.